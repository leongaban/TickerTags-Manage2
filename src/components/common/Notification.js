import React from 'react'
import PropTypes from 'prop-types'
import { tagline } from '../../consts/sentences'

const Notification = ({ message, closeMsg }) => {
	return (
		<div className="notification" onClick={ closeMsg }>
			<div className="message">
				<p>{ message }</p>
				<div className="icon-cancel-circled-1"></div>
			</div>
		</div>
	)
}

Notification.propTypes = {
	message: PropTypes.string.isRequired
}

Notification.defaultProps = {
	message: tagline
};

export default Notification