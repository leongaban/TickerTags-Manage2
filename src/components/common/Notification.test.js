import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Notification from '../common/Notification'

const notificationComponent = shallow(<Notification />);

describe('<Notification /> component', () => {
	it('should render', () => {
		const tree = toJson(notificationComponent);
		expect(tree).toMatchSnapshot();
	});
});