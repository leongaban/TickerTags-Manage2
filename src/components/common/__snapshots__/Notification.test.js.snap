// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Notification /> component should render 1`] = `
<div
  className="notification"
>
  <div
    className="message"
  >
    <p>
      All change is detectable.
    </p>
    <div
      className="icon-cancel-circled-1"
    />
  </div>
</div>
`;
